build/Dockerfile

```diff
@@ -15,31 +15,30 @@ RUN tar -C /usr/local -xzf go.tar.gz
 ENV PATH=${PATH}:/usr/local/go/bin
 
 # Build WMCB
-WORKDIR /build/
-RUN git clone --single-branch --branch release-4.6 https://github.com/openshift/windows-machine-config-bootstrapper.git
-WORKDIR windows-machine-config-bootstrapper
+RUN mkdir /build/windows-machine-config-bootstrapper
```

Comment:
Why is this needed? [WORKDIR](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#workdir) should create the directory.

Reply:
yes, added this change